db/import.go

```diff
@@ -208,7 +208,7 @@ func (db *Database) importDoc(docid string, body Body, isDelete bool, existingDo
 		if err != nil {
 			return nil, nil, updatedExpiry, err
 		}
-		base.InfofCtx(db.Ctx, base.KeyImport, "Created new rev ID for doc %q / %q", base.UD(docid), newRev)
+		base.DebugfCtx(db.Ctx, base.KeyImport, "Created new rev ID for doc %q / %q", base.UD(docid), newRev)
```

Comment:
Was knocking the Import specific logging down a level always in scope for this?

Reply:
I thought in general we intended to reduce per-doc logging which would include imports? Thats why this was knocked down.

Reply:
In general, yeah, but Cache is the much bigger contributor there, and everything we've considered on the stats side has been to replace cache logging. Not saying necessarily that this *isn't* covered by stats, but I don't think we've been through that exercise to check and I'm hesitant to reduce our `info` level visibility of Import to 0.

Reply:
Ok, I'll put this back to info and perhaps we can review this separately

Reply:
@JFlath The intention was to reduce per-doc logging during import, and focus on logging import-related errors.  Information about whether a given doc was successfully imported is available via the current stats of the document in the bucket, so I don't think it makes sense to spam the logs with per-revision logging.

Reply:
I think that's perfectly reasonable as a goal, my only concern is that this PR adds cache stats to replace the cache logging it removed (or reduces), but doesn't add anything on the stats side to balance out *this* removal. Maybe best to spit into two PRs then?

Reply:
I didn't think that was necessary in this case because we already have an import count stat that would cover this item.

Reply:
I think it goes a long way to covering it, but my concern is that it doesn't actually give us the same data as the logs we're dropping down here (or rather, we can't infer the same data). Currently, we have:

```
	// StatsSharedBucketImport
	StatKeyImportCount          = "import_count"
	StatKeyImportCancelCAS      = "import_cancel_cas"
	StatKeyImportErrorCount     = "import_error_count"
	StatKeyImportProcessingTime = "import_processing_time"
```

`import_count` is probably the main one here, but this doesn't give us the visibility that the other stats added by this PR do. We wouldn't be able to narrow down when a given seq was imported.

Something like `high_seq_imported` *would* give us that visibility. I'm cautious of adding creep to this PR, but if we add that then maybe a `high_seq_assigned` would also make sense...

Reply:
Lets get this information into a CBG and then we can merge this PR as-is for the cache logging/stats enhancements.